docs/Ways-of-Working/Toolkit/Test-Engineering/Best-Practices/Playwright-Test-Automation-Guidelines.md

```diff
@@ -0,0 +1,111 @@
+
+For whichever annotation we use, it is imperative that we put a comment above the annotation to justify why it's being used, to ensure it's clear to other team members.
+
+For further examples of annotations playwright support, please follow the Playwright documentation found [here](https://playwright.dev/docs/test-annotations)
+
+**In summary:**
+
+- Use `test.only` to run a single test. (Playwright will only run the test with the `test.only` annotation applied to it).
+
+This should be used in instances where we want to focus on a single test and ignore all other tests. 
+
+**We should **NEVER** merge tests with `test.only` annotations into the main branch, but it has it's uses when executing tests locally or on work / feature branches at times.**
+
+- Use `test.fixme` to mark a test that is known to be failing and requires a fix. (Playwright will not run the test and it will be skipped).
+
+This should be used in instances where the test logic is fine but the test is failing due to a known defect. There should be a BLI in place to fix this within the next 2 sprints and we are aware a fix is in the works. Its imperative we include a comment above the `test.fixme` annotation to explain why the test is failing, along with a reference ID of the DevOps BLI. 
+
+**We can merge tests with `test.fixme` annotations into the main branch, but only temporarily. This should then be reverted back to `test` once the test is fixed.**
+
+- Use `test.skip` to skip a test. (Playwright will not run the test and it will be skipped).
+
+There are 2 main instances why we would use this annotation:
+
+1 - To skip a test temporarily in order to focus on other tests. This can be beneficial when running tests *locally* or on work / feature branches, but shouldnt be merged to main in these instances. 
+
+2 - To skip a work in progress test, whereby the logic in the test would fail at this particular instance in time because it's not ready for testing yet, but we want to keep the test in the codebase for future reference or enhancement.
+
+3 - To skip a test under certain conditions, such as not running the test on a specific browser if it is flaky. For example:
+
+            import { test, expect } from '@playwright/test';
+
+            test('Safari-only test', async ({ page, browserName }) => {
+            test.skip(browserName !== 'webkit', 'This feature is Safari-only');
+            // ...
+            });
+
+4 - To skip a failing test, under instances when we do not have a BLI in place to fix the test and we are unsure of the timframe of the fix. In the instance of a failing test with a known fix in the works, we should use `test.fixme` instead, explained further below.
```

Comment:
Spelling error - timeframe

Reply:
Is it current practice to have BLIs for a broken test?

Reply:
spelling error updated :)

for a skipped test that is failing, then we should have a BLI in place for traceability..... either to be actioned within 2 sprints (test.fixme) which should be on a delivery teams sprint board or on our minmi board (test.skip) when we dont know the timeframe of the fix. have updated the notes.